WordPress/src/main/java/org/wordpress/android/models/AccountModel.java

```diff
@@ -20,6 +23,7 @@
     private String mFirstName;
     private String mLastName;
     private String mAboutMe;
+    private Date mDate;
```

Comment:
Sure! I'm glad you brought that up as I wasn't happy with the naming too. I'll change it :)